lib/ui/painting.dart

```diff
@@ -1737,6 +1753,21 @@ class Image {
     return _image.toByteData(format: format);
   }
 
+  /// The color space that used by the [Image]'s colors. Note that on certain
+  /// platforms/rendering backends, wide gamut images will still report
+  /// [ColorSpace.sRGB] if rendering wide gamut colors isn't supported.
```

Comment:
which platforms/rendering backends? Specificity is the soul of narrative.

Reply:
(in general, when writing documentation, think to yourself "what questions does what i've written raise in the reader's mind?" and then answer those questions as well.)

Reply:
I'm worried that if I say, it will become stale.  There's not a good way to make sure that this get updated as new platforms start handling wide gamut images.  Isn't that a concern?

Reply:
Addressed in another comment, added supported platforms.